src/main/java/DateTime.java

```diff
@@ -0,0 +1,65 @@
+public class DateTime {
+    private String dateTime;
+    private String date;
+    private String time;
+
+    public DateTime(String s) {
+        dateTime = s;
+        String[] arr = s.split(" ");
+        date = arr[1];
+        time = arr[2];
+    }
+
+    public String convertDate() {
+        String[] arr = date.split("/");
```

Comment:
As what podocarp suggested, or what I did was create arrays.. etc monthArr = { th, st, nd, rd ........ } and you can call monthArr[1] = "1st".